data/scripts/dependency-resolver.py

```diff
@@ -395,9 +419,12 @@ def run(args, dep_checks, context):
             exit(1)
 
         result = handler(args, dep, context)
+        context.add_log_content("#############################")
         if verbose:
             print("found." if result else "not found.")
 
+    context.write_log_content(".config.log")
```

Comment:
as above, just log from the beginning. You can move the old log to `.config.log.old` if you want to keep it.